src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+const employeesList = document.querySelectorAll('li');
+
+function sortList(list) {
+  const salaryToFloat = (salary) => (
+    parseFloat(salary.replace('$', '').replace(',', '.'))
+  );
+
+  const sortedList = [...list].sort((a, b) => {
+    const salaryA = salaryToFloat(a.dataset.salary);
+    const salaryB = salaryToFloat(b.dataset.salary);
+
+    return salaryA < salaryB ? 1 : -1;
+  });
+
+  document.querySelector('ul').append(...sortedList);
+}
+
+function getEmployees(list) {
+  return [...list].map(employee => {
```

Comment:
I don't see any sense in destructuring [...list], because the method map() already return new array,
make sense to destructure employee